Automattic/wp-calypso
# Do NOT redirect `/plugins/manage/` multisite view and check for 'install-purchased-plugins' feature.

#### Proposed Changes

Redirects `/plugins/manage/{site_slug}` (If missing 'manage-plugins' or 'install-purchased-plugins') to `/plugins/{site_slug}`

Going to `/plugins/manage/` (with no site_slug) should NOT redirect you anywhere.

Show "Installed Plugins" button on all Jetpack (non-Atomic) sites OR sites with the 'manage-plugins' OR 'install-purchased-plugins' feature. Hide the button for simple and Atomic sites without the 'manage-plugins' OR 'install-purchased-plugins' feature.

![installed-plugins](https://user-images.githubusercontent.com/140841/175573344-c91e977f-2d0c-4a93-afb3-be5439ccd7a4.png)

#### Testing Instructions

> **Note**
> Jetpack (non-Atomic) sites should always show the "Installed Plugins" & "Upload" plugin buttons and never be redirected from `/plugins/manage/{site_slug}` or `/plugins/upload/{site_slug}`


- Load this branch.

For all Simple sites and Atomic WITHOUT the 'manage-plugins' or 'install-purchased-plugins' feature (this is currently all plans below Business and Pro).
- Go to `/plugins/{site_slug}`
  - You should NOT see the "Installed Plugins" button.
- Try manually going to `/plugins/manage/{site_slug}`
  - You should be redirected to /plugins/{site_slug}
- Try manually going to `/plugins/manage/`
  - You should NOT be redirected

For Atomic sites WITH the 'manage-plugins' or 'install-purchased-plugins' feature (this is currently Business, Pro, and eCommerce plans).
* Got to `/plugins/{site_slug}`
  * You should see the "Installed Plugins"
* Try manually going to `/plugins/manage/{site_slug}`
  * You should NOT be redirected to /plugins/{site_slug}

Regression test: For plans on Simple and Atomic WITHOUT the 'upload-plugins' feature (this is currently all plans below Business and Pro).
- Go to `/plugins/{site_slug}`
  - You should NOT see the "Uploads" button.
- Try manually going to `/plugins/upload/{site_slug}`
  - You should be redirected to /plugins/{site_slug}

Regression test: For plans on Simple and Atomic WITH the 'upload-plugins' feature (this is currently Business, Pro, and eCommerce plans).
- Go to `/plugins/{site_slug}`
  - You should see the "Uploads" button.
- Try manually going to `/plugins/upload/{site_slug}`
  - You should NOT be redirected
